src/Microsoft.Health.Dicom.Tests.Common/Comparers/DicomDatasetEqualityComparer.cs

```diff
@@ -0,0 +1,50 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using Dicom;
+using Dicom.Imaging;
+
+namespace Microsoft.Health.Dicom.Tests.Common.Comparers
+{
+    public class DicomDatasetEqualityComparer : IEqualityComparer<DicomDataset>
+    {
+        public DicomDatasetEqualityComparer()
+            : this(new DicomTag[0])
+        {
+        }
+
+        public DicomDatasetEqualityComparer(IEnumerable<DicomTag> ignoredTags) => IgnoredTags = ignoredTags;
+
+        public IEnumerable<DicomTag> IgnoredTags { get; }
+
+        bool IEqualityComparer<DicomDataset>.Equals(DicomDataset x, DicomDataset y)
+        {
+            if (x == null || y == null)
+            {
+                return object.ReferenceEquals(x, y);
+            }
+
+            // Ignore PixelData here, will check later with DicomPixelDataEqualityComparer
```

Comment:
>Ignore PixelData here, will check later with DicomPixelDataEqualityComparer [](start = 15, length = 75)

It probably is sufficient for this comment to say - "First validiate...."

Reply:
will update...